src/data/services/lms/api.js

```diff
@@ -11,9 +11,9 @@ import * as utils from './utils';
 
 const { get, post, stringifyUrl } = utils;
 
-/*********************************************************************************
+/** *******************************************************************************
```

Comment:
this makes me sad... if we have to do this plz, just move down to simple block-comment.